cryptofeed/backends/gcppubsub.py

```diff
@@ -0,0 +1,94 @@
+'''
+Copyright (C) 2017-2020  Bryant Moscon - bmoscon@gmail.com
+
+Please see the LICENSE file for the terms and conditions
+associated with this software.
+'''
+import os
+
+import aiohttp
+import google.api_core.exceptions
```

Comment:
these google imports will need to be added to the `extras` in setup.py. You can list them under something like 'gcp' or 'GCPPubSub' if you like

Reply:
Added as 'gcp_pubsub'